oneview/resource_server_profile.go

```diff
@@ -95,49 +88,36 @@ func resourceServerProfile() *schema.Resource {
 func resourceServerProfileCreate(d *schema.ResourceData, meta interface{}) error {
 	config := meta.(*Config)
 
-	serverProfileTemplate, err := config.ovClient.GetProfileTemplateByName(d.Get("template").(string))
-	if err != nil || serverProfileTemplate.URI.IsNil() {
-		return fmt.Errorf("Could not find Server Profile Template\n%+v", d.Get("template").(string))
+	serverProfile := ov.ServerProfile{
+		Type: d.Get("type").(string),
+		Name: d.Get("name").(string),
 	}
-	var serverHardware ov.ServerHardware
+
 	if val, ok := d.GetOk("hardware_name"); ok {
-		serverHardware, err = config.ovClient.GetServerHardwareByName(val.(string))
+		serverHardware, err := config.ovClient.GetServerHardwareByName(val.(string))
 		if err != nil {
 			return err
 		}
-	} else {
-		var hwFilters = []string{}
```

Comment:
Are we not using `hw_filter` anymore? We'll want to remove it from the schema and documentation if so